client/landing/subscriptions/router/routes/subscriptions/comments/comments.tsx

```diff
@@ -1,4 +1,4 @@
-import TabView from './tab-view';
+import { TabView } from '../../../../components/tab-view';
```

Comment:
Instead of relative paths we could use `calypso` import alias to make imports more readable, like `calypso/landing/subscriptions/components/tab-view`. It applies to other internal component imports as well.

Reply:
Ok. Let's go for it. 😁 I had this dilemma myself. I decided to go for relative, just because I thought that we are sort of a slice of a bigger package, and it would be easier to move our codebase around if the paths were relative within. But yeah, it would be a mess to move it either way, so let's settle for readability as you suggested. 🙂